webapp/client/src/components/FormFieldCheckBox.js

```diff
@@ -0,0 +1,103 @@
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import FormFieldScaffolding from "./FormFieldScaffolding";
+import checkedIcon from "../assets/icons/checked.svg";
+
+const CheckBox = styled.div`
+  & {
+    padding: 0;
+    margin-top: var(--spacing-02);
+    flex-wrap: inherit;
+  }
+
+  input {
+    width: 30px;
+    height: 30px;
+    opacity: 0;
+  }
+
+  input:focus + label {
+    box-shadow: 0 0 0 3px var(--focus-color);
+    background-color: var(--focus-color);
+  }
+
+  input:checked + label {
+    background-color: var(--link-color);
+    background-image: url(${checkedIcon});
+    background-repeat: no-repeat;
+    background-size: 22px;
```

Comment:
I know you didn't add this, I'm just wondering why the background size is 22px and not 30?

Reply:
Because it is the size for the checkbox in components.css?